demo/ios/OrcaDemo/OrcaDemo/ContentView.swift

```diff
@@ -100,12 +142,12 @@ struct ContentView: View {
                         Text(viewModel.state == .PLAYING ? "Stop" : "Synthesize")
                         .padding()
                         .frame(minWidth: 200)
-                        .background(interactionDisabled ? Color.gray : activeBlue)
+                        .background(buttonDisabled ? Color.gray : activeBlue)
                         .foregroundColor(Color.white)
                         .font(.largeTitle)
                     }
                 )
-                .disabled(interactionDisabled)
+                .disabled(buttonDisabled)
```

Comment:
I can't demo this at home right now without my mac, but ensure this demo is almost identical to the Android one, as I've already tried that one out